BE/app/worker.py

```
@@ -1,15 +1,89 @@
+                },
+            )
+            session.add(verified)
+
+            await session.commit()
+            logger.info(f"[DB] 결과 저장 완료 (topic_request_id={topic_request_id})")
+        except Exception as e:
+            await session.rollback()
+            logger.error(f"[DB] 결과 저장 실패: {e}", exc_info=True)
+
+
+async def _create_topic_request(topic: str, user_id: str = None, channel_id: str = None):
```

Ruff (0.14.14)

[warning] PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)